cmds/wget/wget_test.go

```diff
@@ -42,53 +39,38 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 var tests = []struct {
-	flags   []string // in, %[1]d is the server's port, %[2] is an unopen port
-	url     string   // in
-	stdout  string   // out
-	retCode int      // out
+	url    string
+	stdout string
+	isErr  bool
 }{
 	{
 		// basic
-		flags:   []string{},
-		url:     "http://localhost:%[1]d/200",
-		stdout:  content,
-		retCode: 0,
+		url:    "http://localhost:%[1]d/200",
+		stdout: content,
 	}, {
 		// ipv4
-		flags:   []string{},
-		url:     "http://127.0.0.1:%[1]d/200",
-		stdout:  content,
-		retCode: 0,
-	}, /*{ TODO: travis does not support ipv6
+		url:    "http://127.0.0.1:%[1]d/200",
+		stdout: content,
+	}, {
 		// ipv6
-		flags:   []string{},
-		url:     "http://[::1]:%[1]d/200",
-		stdout:  content,
-		retCode: 0,
-	},*/{
+		url:    "http://[::1]:%[1]d/200",
```

Comment:
Does Travis and CircleCI support ipv6 now?

Reply:
Nope. I have to disable it again.